CopyAutoMapperAssembly fails on absolute OutputPath

`@(Content)` item group cannot contain absolute `%(FullPath)` as that'll break MBSuild's ref parser on redirecting.
1. Create a web app project
2. Install latest (3.2.0) AutoMapper nuget, i.e. `Install-Package AutoMapper -Pre`
3. Confirm `WebApplication1.csproj` imports `AutoMapper.targets`, i.e. `<Import Project="..\packages\AutoMapper.3.2.0-ci1010\tools\AutoMapper.targets"`
4. Build project with an absolute redirect, e.g. `msbuild WebApplication1\WebApplication1.csproj /p:OutputPath=%TEMP%\Foo`

```
Build FAILED.

"C:\Users\ilya.kozhevnikov\Dropbox\WebApplication1\WebApplication1\WebApplication1.csproj" (default target) (1) ->
(_CopyWebApplicationLegacy target) -> 
  C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v12.0\WebApplications\Microsoft.WebApplication.targets(182,5): error MSB3021: Unable to copy file "C:\Users\ilya.kozhevnikov\Dropbox\WebApplication1\packages\AutoMapper.3.2.0-ci1010\lib\net40\AutoMapper.Net4.dll" to "C:\Users\ILYA~1.KOZ\AppData\Local\Temp\Foo\_PublishedWebsites\WebApplication1\C:\Users\ilya.kozhevnikov\Dropbox\WebApplication1\packages\AutoMapper.3.2.0-ci1010\lib\net40\AutoMapper.Net4.dll". The given path's format is not supported. [C:\Users\ilya.kozhevnikov\Dropbox\WebApplication1\WebApplication1\WebApplication1.csproj]
```

--- a/src/AutoMapper.targets
+++ b/src/AutoMapper.targets
@@ -13,7 +13,7 @@
     <Message Importance="low" Text="AutoMapper Platform Extension Assembly: %(_AutoMapperReference.FullPath)"/>
 
     <ItemGroup>
-      <Content Include="%(_AutoMapperReference.FullPath)" Condition="'%(_AutoMapperReference.FullPath)' != ''">
+      <Content Include="$([MSBuild]::MakeRelative($(MSBuildProjectDirectory), %(_AutoMapperReference.FullPath)))" Condition="'%(_AutoMapperReference.FullPath)' != ''">
         <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       </Content>
     </ItemGroup>

